### ISSUE TITLE
Screenreaders can read content outside the modals on the results page

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4082]
## Area or Component
Results tab - download results button/modal

(Also probably an issue for the print modal, correct test modal, and the text/email results modals)

## Quick Summary
Screen readers can read content outside of the various modals on the Results page. 

## Accessibility Violation
Able to browse outside modal with screen reader: https://cdc-devtools.dequecloud.com/rule-help/en/reading-order-browse-outside-modal

## Action Requested
Set all other elements on the page to `aria-hidden:true`. See #4075 for a similar issue; this is true of modals across our site.

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=1978534823&range=6:6

**Comment by emmastephenson** [comment]
Note that this may be a false positive - if the element is using `react-modal`, then all other elements should be marked as `aria-hidden` automatically. The only that isn't is the `div` containing the modal, which doesn't have any content so it doesn't matter.

The ask therefore is to validate that this issue is a false positive for all the modals on the result page (CSV download, print result, email result, text result, correct result, and view details.) If those modals use react-modal, there should be no further action to take.

cc @mehansen and @georgehager, who have looked more extensively into these modal issues

**Comment by georgehager** [comment]
Based on research in #4053 and #4102, we believe this is a false positive. Please see those tickets for reference.

We have tested all modals on the results page using NVDA. See screen recordings below:

Additionally, using the Aria DevTools extension, we are able to view the tree that will be read by the screen reader. It does not show any text readable outside the modals:
- Download Results modal
![Screen Shot 2022-09-29 at 4 56 13 PM](https://user-images.githubusercontent.com/89797785/193140526-ae72f906-9443-423e-b8e4-e227aa33cdf9.png)

- View Details modal
![Screen Shot 2022-09-29 at 4 58 44 PM](https://user-images.githubusercontent.com/89797785/193140824-c3074347-c58d-455e-be09-e0bdcefe91b1.png)

- Print Results modal
https://user-images.githubusercontent.com/89797785/193140989-d2135b89-6318-4b2b-8f28-d3ce0b720216.mov

- Text Results modal
![Screen Shot 2022-09-29 at 5 01 10 PM](https://user-images.githubusercontent.com/89797785/193141272-28dd96e7-dae6-46e2-bbd3-e8bf618591a4.png)

- Email Results modal
![Screen Shot 2022-09-29 at 5 00 20 PM](https://user-images.githubusercontent.com/89797785/193141135-49aa0c2e-7ce0-4412-a9b4-1f768cb724cd.png)

- Correct Results modal
![Screen Shot 2022-09-29 at 5 01 54 PM](https://user-images.githubusercontent.com/89797785/193141362-85ad914d-fe15-41f4-90af-7b38783db16f.png)

**Comment by georgehager** [comment]
### NVDA Screen recordings

These are large files, so they are stored in google drive at these links:
- [Download Results](https://drive.google.com/file/d/1SzN9a0mrGRw6HtGjIvec1wmrazWGb6eo/view?usp=sharing)
- [View Details](https://drive.google.com/file/d/1XF7kwDwgY85k3v6VG0i7EZaB4FKPOkOp/view?usp=sharing)
- [Print Results](https://drive.google.com/file/d/1YNsFWZln-zsUu0V725wLbKIbwubETYPQ/view?usp=sharing)
- [Email Results](https://drive.google.com/file/d/1062S1gU6Eb_fHr2brZt7b64f-QPR-Ui2/view?usp=sharing)
- [Text Results](https://drive.google.com/file/d/1GzCPhj54oHBc1nLlm0DW7nyUV8lzmsQ2/view?usp=sharing)
- [Correct Results](https://drive.google.com/file/d/1eWr7F-nYlzEoS-Q3sNTY-dvOQF7bs8gT/view?usp=sharing)

**Comment by georgehager** [comment]
Closing as a confirmed false positive. cc: @emmastephenson
